### DISCUSSION TITLE
[2024-07-11] 회의록

### DISCUSSION THREAD

**Discussion by ggeunn** [finalprojectSinchon/backend#6] [General]
## 프로젝트 예시 자료 분석
 - 모르는거 서로 질문하기

## REST API 설계

- 회원                         김윤재
  - 로그인
  - 회원가입

- 점포     store              김윤재
- 안내소  info               이권한
- 창고      storage         김재중
- 편의시설 facility     정고은
- 이동수단                    전준규
- 비행기                        이지은
   - 체크인카운터
   - 탑승구
   - 수화물수취대
- 정비 maintenance 이권한
- 점검 inspection 정고은
- 승인 approval
- 장비 equipment

**Comment by GonY0624** [comment]
굿 이지은

**Reply by gwonhan** [reply]
갓 지은 밥

**Comment by gwonhan** [comment]
정 신나간
고 은아
은 제 정신차릴래

**Reply by GonY0624** [reply]
이 빠진 
권 한아
한 국을 떠나라
